elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClient.scala

```diff
@@ -183,7 +190,7 @@ class RestlasticSearchClient(endpointProvider: EndpointProvider, signer: Option[
                       endpoint: String,
                       method: HttpMethod = POST,
                       query: UriQuery = UriQuery.Empty)
-                     (implicit ec: ExecutionContext): Future[RawJsonResponse] = {
```

Comment:
I don't think anyone was using this endpoint, but FWIW, this isn't strictly backwards compatible. Having a default value for a second parameter still requires a code change, I think